agent/util-scripts/pbench-tool-meister-start

```diff
@@ -464,7 +353,7 @@ class ToolDataSink(BaseServer):
         redis_server,
```

Comment:
Type hint?

Reply:
Yes, ordering problem and I did not want to blow up the code with a major code move as part of the PR.